classes/mapworker.php

```diff
@@ -289,6 +350,10 @@ public function process_map_objects() : void {
             // Handle unavailable places.
             foreach ($notavailable as $place) {
                 $domplace = $this->dom->getElementById($place);
+                unset($this->coordinates[$place]);
+                for ($i = 1; $i < 5; $i++) {
+                    unset($this->coordinates['text' . $i . $place]);
+                }
```

Comment:
There are four text coordinates? What exactly are these?

Reply:
The four corners of the rectangle containing the text.

Reply:
alright, I just saw there is a comment describing this at the point where they are defined, so that's fine